parse.js

```diff
@@ -0,0 +1,86 @@
+'use strict';
+var privateParse = require('./lib/parse');
+
+/**
+ * Functions for parsing and analysing dependencies of AMD modules. Mostly a
+ * wrapper around lib/ modules, but want to try not exposing the libs directly,
+ * and provide a cleaner interface.
+ */
+var parse = {
+  /**
+   * Parses the input JavaScript text and returns a AST of it. Expects a
+   * JavaScript content. The returned object can be used later in other calls,
+   * or to other module analysis.
+   *
+   * @param {String} contents File contents of an AMD module.
+   * @param {Object} options Optional. Options for the `esprima` parser: Only
+   * `range` and `loc` properties are recognized.
+   * @return {Object} Optional. Options for the `esprima` parser: Only `range`
+   * and `loc` properties are recognized.
+   */
+  parse: function (contents, options) {
+    return privateParse.parseFileContents('', contents, options);
+  },
```

Comment:
See comment about `parseFileContents`, possible removal.

Reply:
Yes, I'm removing it. And the documentation and tests too.